tests/JustSaying.IntegrationTests/Fluent/MessagingBusBuilderTests.cs

```diff
@@ -22,16 +24,20 @@ public async Task Can_Create_Messaging_Bus_Fluently_For_A_Queue()
         var queueName = Guid.NewGuid().ToString();
 
         // Arrange
+        var bus = new InMemoryAwsBus();
         var services = new ServiceCollection()
             .AddLogging((p) => p.AddXUnit(OutputHelper))
             .AddJustSaying(
                 (builder) =>
                 {
                     builder.Client((options) =>
-                            options.WithBasicCredentials("accessKey", "secretKey")
-                                .WithServiceUri(TestEnvironment.SimulatorUrl))
+                            options.WithClientFactory(() => new LocalAwsClientFactory(bus))
+                            // TODO Add back LocalStack config for running in CI
+                            // options.WithBasicCredentials("accessKey", "secretKey")
+                            //     .WithServiceUri(TestEnvironment.SimulatorUrl)
+                            )
```

Comment:
Happy to keep these TODO's and discuss later if we want to action or remove them